.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -58,9 +53,9 @@ jobs:
       KATA_HOST_OS: ${{ matrix.host_os }}
       KATA_HYPERVISOR: ${{ matrix.vmm }}
       # Set to install the KBS for attestation tests
-      KBS: ${{ matrix.kbs }}
+      KBS: ${{ (matrix.vmm == 'qemu' && matrix.host_os == 'ubuntu') && 'true' || 'false' }}
```

Comment:
Does this condition really depends on host_os? As far as I understood the previous version only cared about vmm...

Reply:
Hi @ldoktor ,

You are right, in the previous version I wasn't checking host_os. I thought in use that variable too to really restrict further the jobs that will have KBS installed. With the old condition, if someone adds qemu vmm to the mariner configuration than it would enable attestation to that combination, which we don't want (not at this point at least).

BTW, perhaps we should enable KBS on only one instance_type too...

Reply:
I'd go with no unnecessary conditions and I'd leave it on those who add the new types to decide. Anyway either way is fine by me.

Reply:
hi @ldoktor , I kept the most restrictive condition.